src/lib/components/tabs/QTab.scss

```diff
@@ -0,0 +1,71 @@
+@use "$css/mixins";
+
+.q-tab {
+  position: relative;
+  box-sizing: border-box;
+  min-height: 3rem;
```

Comment:
Do we need both `min-height` and `height`?

Reply:
I think so, we don't want the browser to shrink them smaller than that